test/cli/commands_test.py

```diff
@@ -522,6 +523,7 @@ def test__cli__command_lint_serialize_multiple_files(serialize):
         raise Exception
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="Paths won't match test on windows")
```

Comment:
Looks fine. Would it be possible (and worthwhile) to modify the test so it works on Windows as well?

Reply:
Heh, I think I wrote this test. I'll take a quick look to see about Windows compatibility.

Reply:
Don't stress - I think it's a great test for unix style platforms - which are also the main targets for it, especially given the github action. I'm fine just skipping it for windows unless we fancy working out a way of properly normalising both sides of the comparison.